{
  "cron-validation/invalid-timezone": "The timezone {cronTimezone} is currently not supported. Please chose another timezone.",
  "dbtcloud/access-denied": "dbt Cloud denied access. Please verify your service token and access URL.",
  "auth/user-already-exists": "A user with the email {email} already exists.",
  "generate-contribution": "An error occurred while generating the contribution: {message}",
  "generate-contribution/invalid-github-token": "The GitHub token provided is invalid. Please verify the token and try again.",
  "generate-contribution/insufficient-github-token-permissions": "We could not generate your contribution on GitHub. Please provide an access token with the \"public_repo\" scope enabled.",
  "generate-contribution/connector-image-name-in-use": "The Connector Identifier you provided is already in use. Please use a different identifier."
}
